### PR TITLE
Calum's shitty PR

### PR DESCRIPTION
**Pull request by X-yl**

### CODE REVIEW

**Comment by X-yl** [comment]
wait fuck I've fucked this up

**Comment by X-yl** [comment]
fixed it nvm

File: assets/css/intro.css [hunk 1]
```diff
@@ -0,0 +1,6 @@
+@import url('https://fonts.googleapis.com/css2?family=Teko:wght@300;400;700&display=swap');
+
+.tagline {
+    font-family: 'Teko';
+    font-size: 75px;
+}
```
Comment (X-yl) [inline assets/css/intro.css:6]: newline

File: assets/css/navbar.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    -webkit-backdrop-filter: blur(8px);  /* Safari 9+ */
+    backdrop-filter: blur(5px); /* Chrome and Opera */
+    box-shadow: 0px 10px 15px 10px rgba(0, 0, 0, 0.09);
+    background-color: rgba(228, 228, 228, 0.052); 
+}
+  
+#jumbotron {
+    display: flex;
+    flex-direction: row;
+}
```
Comment (X-yl) [inline assets/css/navbar.css:64]: newline

File: assets/css/navbar.css [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    position: absolute;
+    font-size: 13px;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: 10px;
+    justify-content: space-between;
+    width: 100%;
+
+    -webkit-backdrop-filter: blur(8px);  /* Safari 9+ */
+    backdrop-filter: blur(5px); /* Chrome and Opera */
```
Comment (X-yl) [inline assets/css/navbar.css:56]: doesn't work on firefox

File: assets/css/style.css [hunk 4]
```diff
@@ -43,7 +42,7 @@ h1 {
   display: flex;
   justify-content: center;
   align-items: center;
-  flex-direction: column;
+  flex-direction: row;
 }
 
 #seds-logo {
```
Comment (X-yl) [inline assets/css/style.css:48]: make smaller pls

the seds logo I mean

**Review by X-yl — COMMENTED** [review]
Also fix your dumb commit message. @calum-s 

for some reason we're now adopting good collaboration practices.

**Comment by calum-s** [comment]
> _ No description provided. _

most professional github collaboration

**Comment by cakeforcat** [comment]
pls let Calum merge

**Pull request merged**